src/modules/localization/src/frame_tf_calibrate/frame_calibration.cpp

```diff
@@ -150,13 +182,37 @@ namespace frame_calibration_node
                 geodesy_tf_msg.pose.pose.position.x = geodesy_tf_point.point.x;
                 geodesy_tf_msg.pose.pose.position.y = geodesy_tf_point.point.y;
                 geodesy_tf_msg.header.frame_id = geodesy_tf_point.header.frame_id;
+                // geodesy_tf_msg.child_frame_id = temp_geodesy_tf_point.header.frame_id;
                 geodesy_tf_msg.header.stamp = ros::Time::now();
 
-                geodesy_tf_pub.publish(geodesy_tf_msg);
+                // counter to remove edge case of first data point

```

Comment:
If this is its only purpose, maybe consider replacing with boolean for clarity and to save memory